Dockerfile

```diff
@@ -0,0 +1,31 @@
+# syntax=docker/dockerfile:1
+
+# Build
+FROM golang:1.20-alpine AS build
+
+WORKDIR /app
+
+COPY go.mod ./
+COPY go.sum ./
+RUN go mod download
+
+COPY *.go ./
+
+RUN go build -o /img-LinuxFr.org
+
+RUN go install golang.org/x/vuln/cmd/govulncheck@latest
+RUN govulncheck /img-LinuxFr.org
+#RUN govulncheck ./...
+
+# Deploy
+FROM alpine
+USER 1000
+
+WORKDIR /
+
+COPY --from=build /img-LinuxFr.org /img-LinuxFr.org
+
+EXPOSE 8000
+
+
+ENTRYPOINT ["/img-LinuxFr.org"]
```

Comment:
For your information, there's an option `-r` to link the service with a Redis server.

Maybe you'll want that Redis server to debug the image service behavior.

For example, in the Linuxfr.org repository, we use this command:

```Dockerfile
CMD ["sh", "-c", "exec ./img-LinuxFr.org -a 0.0.0.0:8000 -r \"${REDIS_URL##redis://}\""]
```

and a docker-compose file like this one:

```yaml
---
services:

  linuxfr-img:
    build:
      context: .
    environment:
      - REDIS_URL=redis://redis:6379/0
    depends_on:
      - redis
    volumes:
      - cache-img:/img-LinuxFr.org/cache

  redis:
    image: redis:5
    volumes:
      - data-redis:/data

volumes:
  data-redis:
    driver: local
  cache-img:
    driver: local

```

Reply:
done